Doc/library/urllib.parse.rst

```diff
@@ -312,6 +312,10 @@ or on combining URL components into a URL string.
    ``#``, ``@``, or ``:`` will raise a :exc:`ValueError`. If the URL is
    decomposed before parsing, no error will be raised.
 
+   Following the specification in WHATWG which updates RFC 3986, ASCII newline
```

Comment:
hyperlink "the specification in WHATWG" to https://url.spec.whatwg.org/#concept-basic-url-parser.